<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RFamp</class>
 <widget class="QDialog" name="RFamp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbRFamp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>RF quad</string>
   </property>
   <widget class="QTabWidget" name="tabRFquad">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>256</width>
      <height>249</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabRFsettings">
     <attribute name="title">
      <string>RF settings</string>
     </attribute>
     <widget class="QCheckBox" name="chkSRFAENA_ON_OFF">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency, Hz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leGFRAVPPB">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>155</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFALEV">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>105</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbSRFAMOD_CLOSED">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Closed loop</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leGRFAPWR">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFADRV">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>185</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF pwr, Watts</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Drive, %</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbSRFAMOD_OPEN">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Open loop</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF- Vp-p</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFAFREQ">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>55</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Setpoint, Vp-p</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>135</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF+ Vp-p</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leGRFAVPPA">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMassFilter">
     <attribute name="title">
      <string>Mass filter</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>32</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ro, mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>84</y>
        <width>96</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Resolving DC, V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>108</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pole Bias, V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>136</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Resolution, AMU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>160</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>m/z</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFAR0">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFARDC">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>82</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFAPB">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>108</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFARES">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>134</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFAMZ">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbRFAupdate">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>185</y>
        <width>221</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSRFAK">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>56</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>56</y>
        <width>109</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RDC=RFpp/k, k=</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkSDCPWR_ON_OFF">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>176</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DC power supply enable</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRFamp">
     <attribute name="title">
      <string>RF amp</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DC V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DC I in</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>52</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DR pwr in,W</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temp, C</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF Vrms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>126</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF Irms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF fwd pwr,W</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>174</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RF rev pwr,W</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>196</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SWR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>5</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>28</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>52</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>76</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>124</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>148</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>172</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelRRFAAMP_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>196</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
